### ISSUE TITLE
Betterbird and quickfolders

### ISSUE THREAD

**Issue by jjroper** [RealRaven2000/QuickFolders#261]
According to the betterbird web page, all thunderbird extensions always work in betterbird. However, when I open Betterbird, the quickfolders do not appear, so I go to the extension settings and turn it off and back on, and voilà, they appear. That's the whole glitch.

**Comment by RealRaven2000** [comment]
> According to the betterbird web page, all thunderbird extensions always work in betterbird. However, when I open Betterbird, the quickfolders do not appear, so I go to the extension settings and turn it off and back on, and voilà, they appear. That's the whole glitch.

I assume you are using 91.* ?

try to remove and reinstall QuickFolders that usually fixes Add-on cache issues.

**Comment by jjroper** [comment]
> > According to the betterbird web page, all thunderbird extensions always work in betterbird. However, when I open Betterbird, the quickfolders do not appear, so I go to the extension settings and turn it off and back on, and voilà, they appear. That's the whole glitch.
> 
> I assume you are using 91.* ?
> 
> try to remove and reinstall QuickFolders that usually fixes Add-on cache issues.

91.8.1 bb3 and I just installed it, and when I restarted it after install, same issue, same solution.

**Comment by RealRaven2000** [comment]
when you start BB, is there anything useful in JS error console? (Ctrl+Shift+J)

**Comment by jjroper** [comment]
Is this useful?
```
[Exception... "Component returned failure code: 0x80070057
(NS_ERROR_ILLEGAL_VALUE) [nsIObserverService.removeObserver]"  nsresult:
"0x80070057 (NS_ERROR_ILLEGAL_VALUE)"  location: "JS frame ::
***@***.***!/chrome/content/api/NotifyTools/implementation.js
:: onShutdown :: line 115"  data: no] implementation.js:115:18
    onShutdown
***@***.***
!/chrome/content/api/NotifyTools/implementation.js:115
    ExtensionAPI resource://gre/modules/ExtensionCommon.jsm:359
    wrapper resource://gre/modules/ExtensionCommon.jsm:303
    emit resource://gre/modules/ExtensionCommon.jsm:330
    emit resource://gre/modules/Extension.jsm:2147
    shutdown resource://gre/modules/Extension.jsm:2843
    shutdown resource://gre/modules/Extension.jsm:1894
    callBootstrapMethod resource://gre/modules/addons/XPIProvider.jsm:1834
    _shutdown resource://gre/modules/addons/XPIProvider.jsm:1963
    InterpretGeneratorResume self-hosted:1482
    AsyncFunctionNext self-hosted:692
```

**Comment by RealRaven2000** [comment]
> Is this useful?

Not really, no :( I have no idea what happens, maybe enable debug mode in QuickFOlders (advanced settings page at the bottom) . also right-click debug and switch on 

extensions.quickfolders.debug.notifications=true

![image](https://user-images.githubusercontent.com/6314674/165839222-e8f5d927-e858-4e40-be77-1e9deae4b120.png)

**Comment by RealRaven2000** [comment]
Is this behavior still the same? The latest version of QuickFolders is [5.12.1](https://addons.thunderbird.net/en-US/thunderbird/addon/quickfolders-tabbed-folders/versions/5.12.1), does it show up in BB?

**Comment by jjroper** [comment]
I just updated to the most recent BB version and, after it opened,
quickfolders did not appear. It did, once I turned it off and on again.

If you'd like me to test anything, just let me know.
Jim
